ert3/storage/_storage.py

```diff
@@ -2,16 +2,18 @@
 from pathlib import Path
 from typing import Any, Dict, Iterable, Optional, Set
 import io
-
+import logging
 import pandas as pd
 from pydantic import BaseModel
 import requests
 import ert3
 
+from ert_shared.storage.connection import get_info
+
+logger = logging.getLogger(__name__)
 
-_STORAGE_URL = "http://localhost:8000"
-_DATA = "__data__"
-_PARAMETERS = "__parameters__"
+_STORAGE_TOKEN = None
```

Comment:
Should we add an issue on storing these global variables within `.ert` instead?

Reply:
I am not sure if this is ready to be stored in .ert as it changes if ert-storage is restarted, at the moment

Reply:
There should be some discussion as to what information about ert-storage can be assumed to persist

Reply:
I agree, but perhaps that discussion is good to document in an issue? :)

Reply:
https://github.com/equinor/ert/issues/1761